README.md

```diff
@@ -44,4 +44,4 @@ To build the basic setup and upload to your remote destination:
 
     ./build.sh -u
 
-End
+End of File
```

Comment:
I just needed a change while testing :)

By the way, it seems like current/updated shellcheck actions are actually run when you keep pushing.

Reply:
if you make changes to PR, then shellcheck runs on PR